ml-agents/mlagents/trainers/tests/test_meta_curriculum.py

```diff
@@ -102,7 +102,7 @@ def test_get_config():
         hidden_units: 128
         lambd: 0.95
         learning_rate: 5.0e-3
-        max_steps: 300
+        max_steps: 100
```

Comment:
Why does the max_step need to change ?

Reply:
Reduce the amount of time taken for this test. I can break this out into a separate PR but it was needed to get the tests to pass for this one. 

Basically depending on the seed there was a chance this test would not complete an episode. So some PRs it passed and others it didn't. I'll branch this out into a separate PR.